modules/nf-core/bcftools/concat/main.nf

```diff
@@ -2,13 +2,13 @@ process BCFTOOLS_CONCAT {
     tag "$meta.id"
     label 'process_medium'
 
-    conda (params.enable_conda ? "bioconda::bcftools=1.15.1" : null)
+    conda "bioconda::bcftools=1.16"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/bcftools:1.15.1--h0ea216a_0':
-        'quay.io/biocontainers/bcftools:1.15.1--h0ea216a_0' }"
+        'https://depot.galaxyproject.org/singularity/bcftools:1.16--hfe4b78e_1':
+        'quay.io/biocontainers/bcftools:1.16--hfe4b78e_1' }"
 
     input:
-    tuple val(meta), path(vcfs), path(tbi)
+    tuple val(meta), path(vcfs, stageAs: "?/*"), path(tbi, stageAs: "?/*")
```

Comment:
check the `stageAS`...

Reply:
@matthdsm was that you?

Reply:
this might make for a weird header, do we really need it, can we patch it otherwise. Since we take good care of naming, this is has never been an issue for us

Reply:
Wasn't me this time!

Reply:
I'm surprised this even works, wouldn't this stage the file and the index in different dirs?

Reply:
It was @nvnieuwk though, so it came from the same corner, but was signed off on by @pinin4fjords !

Reply:
> this might make for a weird header, do we really need it, can we patch it otherwise. Since we take good care of naming, this is has never been an issue for us
> 

I think we should fix the modules

Reply:
> > this might make for a weird header, do we really need it, can we patch it otherwise. Since we take good care of naming, this is has never been an issue for us
> 
> I think we should fix the modules

agreed

Reply:
<a href="https://imgflip.com/i/74xzio"><img src="https://i.imgflip.com/74xzio.jpg" title="made at imgflip.com"/></a><div><a href="https://imgflip.com/memegenerator">from Imgflip Meme Generator</a></div>

Reply:
It was me indeed!!! mainly to fix some issues with the `vcf_gather_bcftools` module when there were file collisions :p

Reply:
> I'm surprised this even works, wouldn't this stage the file and the index in different dirs?

The vcfs and their indices are in the same folder :)

Reply:
```
$ tree /home/nvnieuwk/Documents/cmgg/nf-cmgg-germline/work/dc/6b334c0bf4012a3729fa8ab2b8cc14
/home/nvnieuwk/Documents/cmgg/nf-cmgg-germline/work/dc/6b334c0bf4012a3729fa8ab2b8cc14
├── 1
│   ├── NA24385D2_NVQ_034_2.g.vcf.gz -> /home/nvnieuwk/Documents/cmgg/nf-cmgg-germline/work/e8/d5799d0ffc28e8e6c02dd46d6519ba/NA24385D2_NVQ_034_2.g.vcf.gz
│   └── NA24385D2_NVQ_034_2.g.vcf.gz.tbi -> /home/nvnieuwk/Documents/cmgg/nf-cmgg-germline/work/e8/d5799d0ffc28e8e6c02dd46d6519ba/NA24385D2_NVQ_034_2.g.vcf.gz.tbi
├── 2
│   ├── NA24385D2_NVQ_034_1.g.vcf.gz -> /home/nvnieuwk/Documents/cmgg/nf-cmgg-germline/work/35/5fdd2da0e67b265f2d7fd245920929/NA24385D2_NVQ_034_1.g.vcf.gz
│   └── NA24385D2_NVQ_034_1.g.vcf.gz.tbi -> /home/nvnieuwk/Documents/cmgg/nf-cmgg-germline/work/35/5fdd2da0e67b265f2d7fd245920929/NA24385D2_NVQ_034_1.g.vcf.gz.tbi
├── 3
│   ├── NA24385D2_NVQ_034_3.g.vcf.gz -> /home/nvnieuwk/Documents/cmgg/nf-cmgg-germline/work/0c/2c83ea186004da76153a9319ac5040/NA24385D2_NVQ_034_3.g.vcf.gz
│   └── NA24385D2_NVQ_034_3.g.vcf.gz.tbi -> /home/nvnieuwk/Documents/cmgg/nf-cmgg-germline/work/0c/2c83ea186004da76153a9319ac5040/NA24385D2_NVQ_034_3.g.vcf.gz.tbi
├── NA24385D2_NVQ_034.g.vcf.gz
└── versions.yml
```

Reply:
would `(vcfs, stageAs: "{$meta.id/*}"), path(tbi, stageAs: "{$meta.id/*}")` work? 🤔

Reply:
> would `path(vcfs, stageAs: "{$meta.id/*}"), path(tbi, stageAs: "{$meta.id/*}")` work? thinking

Won't that just have the same effect as not using `stageAs`? since most filenames are the same as the `meta.id`?

Reply:
`{meta.id_?/*}` or something then? 🤷🏻

Reply:
That would work I think :) @maxulysse and @FriederikeHanssen what do you think?

Reply:
how do you have collisions with `vcf_gather_bcftools`?

Reply:
When the scattered input files have the same name instead of having a unique number in the name

Reply:
Yeah I do that too in my pipeline, but I thought it would be good to also let it work if a pipeline developer doesn't do it this way

Reply:
not sure

Reply:
I think it's a better practice to do it that way, and we should enforce it, people can always patch modules locally if they want to do so.

Reply:
Yeah fine by me!

Reply:
yep I would actually like to rather enforce explicit naming here. When you end up with issues with it later on, it is much harder to backtrack which of the 150 interval files failed.